themes/base.theme.bash

```diff
@@ -79,10 +79,10 @@ function scm {
 }
 
 function scm_prompt_char {
-  if [[ -z $SCM ]]; then scm; fi
```

Comment:
Instead of changing all of these individually, wouldn't it make sense to ensure that `$SCM` always has a value?

What you've done might work, but looks like unnecessary repetition to me - wouldn't initializing `$SCM` with a default value (`$SCM_NONE`?) be easier?